server/test/services/program/BlockDefinitionTest.java

```diff
@@ -193,4 +193,19 @@ private BlockDefinition makeBlockDefinitionWithQuestions() {
             .build();
     return block;
   }
+
+  @Test
+  public void hasNullQuestion_isTrue() {
+    QuestionDefinition nullQuestion = testQuestionBank.nullQuestion().getQuestionDefinition();
+
+    BlockDefinition blockDefinition =
+        BlockDefinition.builder()
+            .setId(9999L)
+            .setName("Block Name")
+            .setDescription("Block Description")
+            .addQuestion(ProgramQuestionDefinition.create(nullQuestion, Optional.empty()))
+            .build();
+
+    assertThat(blockDefinition.hasNullQuestion()).isTrue();
```

Comment:
**nitpick:** add an assertion for it returning false

Reply:
done